src/components/Patient/FileUploadPage.tsx

```diff
@@ -15,6 +17,9 @@ export default function FileUploadPage(props: {
     pathParams: { id: patientId },
     prefetch: !!patientId,
   });
+  if (patient?.is_active === false) {
+    navigate(`/facility/${facilityId}/patient/${patientId}`);
+  }
```

Comment:
This redirect is not right. This component is used by Consultation Files Page and Patient Files Page. But the redirect assumes it was opened by patient files page.

Reply:
Should I remove it? or what's the expected functionality of that redirect?

Reply:
Let's keep thing simple... We'll anyways have permission denied notification if users visit url directly, and we have back button in breadcrumbs to take them to a parent page.

So disabling the button alone should be enough

Reply:
Got it, will update the PR accordingly